tensorrt_llm/_torch/compilation/backend.py

```diff
@@ -19,16 +20,21 @@ class Backend:
 
     _custom_pass_instances: List[PatternMatcherPass] = None
 
-    def __init__(self, enable_inductor=True, enable_userbuffers=False) -> None:
+    def __init__(self,
+                 enable_inductor=True,
+                 enable_userbuffers=False,
+                 enable_multi_stream=True) -> None:
         super().__init__()
         self.elapsed_time = 0
         self.module_inference_event = []
         self.module_inference_time = 0
         self.call_count = 0
         self.custom_passes = Backend.get_custom_pass(enable_userbuffers)
         self.rank = tensorrt_llm.mpi_rank()
-        self.enable_inductor = enable_inductor
-
+        # If multi-stream is enabled, we disable inductor since they are not compatible
+        self.enable_inductor = enable_inductor if enable_multi_stream else False
```

Comment:
Is the condition wrong? Should be
```suggestion
        self.enable_inductor = False if enable_multi_stream else enable_inductor
```

Reply:
Fixed